modules/nw-service-externalip-create.adoc

```diff
@@ -6,20 +6,40 @@
 [id="nw-service-externalip-create_{context}"]
 = Attaching an ExternalIP to a service
 
-You can attach an ExternalIP to a service. If your cluster is configured to allocate an ExternalIP automatically, you might not need to manually attach an ExternalIP to the service.
+You can attach an ExternalIP resource to a service. If your cluster is configured to allocate an ExternalIP automatically, you might not need to manually attach an ExternalIP to the service.
```

Comment:
ExternalIP is a resource but does a customer need to create the resource somewhere?